getting_started: minor copyediting (#253)
r-devel/rdevguide: chapters/getting_started.qmd

-   The development release (`r-devel`) : continually developed version moving from r-release to next major/minor version (x + 1).0.0 or x.(y + 1).0 a few weeks before release (at the start of the "GRAND FEATURE FREEZE").

> The source code of released versions of R can be found at [R/tags](https://svn.r-project.org/R/tags/ "svn release source code folder"), and the patched versions are at [R/branch](https://svn.r-project.org/R/branches/ "svn patched source code folder").

Reviewer: ```suggestion
The source code of released versions of R can be found at [R/tags](https://svn.r-project.org/R/tags/ "svn release source code folder") and the patched versions are at [R/branch](https://svn.r-project.org/R/branches/ "svn patched source code folder").
```

Here are the basic steps intended as a checklist.
For complete instructions please see the section in [R-admin](https://cran.r-project.org/doc/manuals/r-devel/R-admin.html#Installing-R-under-Unix_002dalikes).

> 0. Retrieve R source code via into `TOP_SRCDIR` (note that we retrieve the `r-devel` source code):

Reviewer: ```suggestion
0. Retrieve R source code into `TOP_SRCDIR` (note that we retrieve the `r-devel` source code):
```

    svn update
    ```

>     You can also use a SVN client such as TortoiseSVN (<https://tortoisesvn.net/>, command line tool and Windows Explorer integration) or SlikSVN (<https://sliksvn.com/download/>, just the command line tool) so that it can be also found by other tools.

Reviewer: The last part of this sentence doesn't make much sense so we can improve that at the same time:
```suggestion
    You can also use an SVN client such as TortoiseSVN (<https://tortoisesvn.net/>, a command line tool and Windows Explorer integration) or SlikSVN (<https://sliksvn.com/download/>, a command line tool) to check out and work with the SVN repository.
```
